### PR TITLE
Uncrush Bob's Soul: Complement is not the same as inverse

### PR DESCRIPTION
**Pull request by michaelmalonenz**
When calculating percentages, it's an easy trap to fall into that the
inverse of the percentage is the same as the complement - we (especially
those of us used to finite math) tend to think of 100 as the fully
composed value that will wrap.  Except that when we're estimating the
amount left of something, based on a linear progression, we should use
the complement of the completed percentage to get the remaining and
multiply the result by the existing time expended.

In short, Bob's soul was being unnecessarily (or at least overly
harshly) crushed.  The book is currently estimated to complete in May,
not in 2023.

### CODE REVIEW

**Comment by munificent** [comment]
!!!

I feel like a bird in the wind!
